sdk/src/Core/Amazon.Runtime/IClientConfig.cs

```diff
@@ -36,6 +36,7 @@ public partial interface IClientConfig
         /// By default it is set to false.
         /// </summary>
         bool IgnoreConfiguredEndpointUrls { get; }
+
```

Comment:
I also added a new line where there wasn't a space between properties where I noticed them while I was at it.